### PR TITLE
Fixed a bug in distributed search.

### PR DESCRIPTION
**Pull request by mosuka**
### Context / purpose
Fixed a bug in distributed search.

### Description
See #238 

### Checklist
*Remove or complete this list if required.*
- [x] tested locally
- [x] added unit tests
- [x] included documentation

### CODE REVIEW

File: quickwit-search/src/root.rs [hunk 1]
```diff
@@ -103,6 +104,7 @@ pub async fn root_search(
             split_uris: jobs.iter().map(|job| job.split.clone()).collect(),
```
Comment (fulmicoton) [inline quickwit-search/src/root.rs:104]: so these are split_id, not split_uris right?

We should probably rename?

**Review by fulmicoton — CHANGES_REQUESTED** [review]
I think split_uris should be renamed split_ids

Can you open a ticket to add unit tests?
It is quite difficult, but we should be able to test root and leaf independently via mockall. (I did so in barrel)

File: quickwit-search/src/root.rs (hunk 1 above)
Comment (mosuka) [reply quickwit-search/src/root.rs:104]: Yes, I’ll rename to split_ids.

**Comment by mosuka** [comment]
I created an issue for adding a unit test for distributed search.
https://github.com/quickwit-inc/quickwit/issues/240

**Review by fulmicoton — APPROVED** [review]

**Comment by mosuka** [comment]
@fulmicoton 
Thanks.

**Pull request merged**